libvirt/image/Makefile

```diff
@@ -120,9 +125,32 @@ $(PAUSE_SRC): $(SKOPEO_SRC)/bin/skopeo
 $(PAUSE): | $(PAUSE_SRC) $(UMOCI_SRC)/umoci
 	$(UMOCI_SRC)/umoci unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
 
+docker-build:
+	docker build -t caa-libvirt-ubuntu-podvm-builder \
+	-f Dockerfile_ubuntu.podvm_builder .
+	docker run --rm \
+		-v $(PROJECT_DIR):/src/cloud-api-adaptor \
+		-v $(abspath $(PROJECT_DIR)/../kata-containers):/src/kata-containers \
+		-e UID=$(shell id -u) \
+		-e GID=$(shell id -g) \
+		caa-libvirt-ubuntu-podvm-builder \
+		bash -c 'trap "chown -R -f "$$UID:$$GID" output staticlib $(SKOPEO_SRC) \
+		$(UMOCI_SRC) $(PAUSE_SRC) $(FILES_DIR)/$(PAUSE_BUNDLE) $(FILES_DIR)/usr \
+		../../agent-protocol-forwarder || true" EXIT; \
+		source ~/.cargo/env && CLOUD_PROVIDER=libvirt make build'
```

Comment:
I hit some error running this line as part of `make docker-build`:
```
docker run --rm \
	-v /root/go/src/github.com/wainersm/cc-cloud-api-adaptor:/src/cloud-api-adaptor \
	-v /root/go/src/github.com/wainersm/kata-containers:/src/kata-containers \
	-e UID=0 \
	-e GID=0 \
	caa-libvirt-ubuntu-podvm-builder \
	bash -c 'trap "chown -R -f "$UID:$GID" output staticlib skopeo \
	umoci pause files/pause_bundle files/usr \
	../../agent-protocol-forwarder || true" EXIT; \
	source ~/.cargo/env && CLOUD_PROVIDER=libvirt make build'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  5394  100  5394    0     0  54484      0 --:--:-- --:--:-- --:--:-- 54484
/bin/sh: 1: virsh: not found
```
Is that something to worry about?

Reply:
It is the `ifdef` in the `push` target that is evaluated always. I re-wrote the target to fix it.

Reply:
No virsh error anymore. Thanks!